### PR TITLE
media : modify tc for media recorder

### PR DESCRIPTION
**Pull request by jsdosa**
modified it with tc which does not use gmock and gtest

Signed-off-by: jaesick.shin <jaesick.shin@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [8bf075fb912880780516f09fe39e73f9a1f29807] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8bf075fb912880780516f09fe39e73f9a1f29807] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9f087a6848b932ea1fd752a081f455fecbda87cb] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9f087a6848b932ea1fd752a081f455fecbda87cb] - Code Rule Check (C++) OK.

File: apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp [hunk 1]
```diff
@@ -17,31 +17,376 @@
...
 #include "tc_common.h"
 
-static void utc_media_mediarecorder_create_p(void)
+using namespace std;
+using namespace media;
+using namespace media::stream;
+
+static unsigned char channels = 2;
+static unsigned int sampleRate = 16000;
+static int pcmFormat = 0;
```
[the remark this answers is not in the document]
Comment (jsdosa) [reply apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp]: @junmin-kim ok. thanks!

**Pull request merged**